persistence/sql/pom.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+   <modelVersion>4.0.0</modelVersion>
+   <parent>
+      <groupId>org.infinispan</groupId>
+      <artifactId>infinispan-persistence-parent</artifactId>
+      <version>13.0.0-SNAPSHOT</version>
+      <relativePath>../pom.xml</relativePath>
+   </parent>
+
+   <artifactId>infinispan-cachestore-sql</artifactId>
+   <name>Infinispan SQL CacheStore</name>
+   <description>Infinispan SQL CacheStore module</description>
+
+   <dependencies>
+      <dependency>
+         <groupId>org.infinispan.protostream</groupId>
+         <artifactId>protostream-processor</artifactId>
+      </dependency>
+
+      <dependency>
+         <groupId>com.fasterxml.jackson.core</groupId>
+         <artifactId>jackson-core</artifactId>
+      </dependency>
```

Comment:
It is sadly to do any conversions between json and other types as was discussed in https://infinispan.zulipchat.com/#narrow/stream/118645-infinispan/topic/sql.20store/near/247707385 because of https://github.com/infinispan/protostream/blob/main/core/src/main/java/org/infinispan/protostream/impl/JsonUtils.java#L51